### DISCUSSION TITLE
Hybrid graphics, Wine, Steam, Lutris setup

### DISCUSSION THREAD

**Discussion by volfpeter** [elementary/streams#34] [Ideas]
How to get all these working correctly on top of a clean elementary install. Currently users need to look this up on stackexchange, reddit, etc.

**Comment by cassidyjames** [comment]
Hm… I think this might be a bit outside of the purpose of these streams, but it could be interesting for a community member to take on.

**Comment by danirabbit** [comment]
Yeah I agree this is probably out of scope. Our endorsed method for sideloading apps is from Flatpak. I don't think we really want to promote something like Wine. And hybrid graphics just needs work done to be supported. I don't think we want to encourage people to take on hacks and tweaks

**Reply by rasmus91** [reply]
I'd be inclined, in the case of wine, to encourage whoever looks into that, to look into making flatpak apps with a wine setup, that just needs the gamefiles placed in some specific folder, or with a script that imports game files upon first run. That would be somewhat similar to what Valve is doing on steam with pressure vessel, and that way the setup will be pretty system agnostic.

But yes, I too would not encourage supporting hacks and tweaks, that is a rabbithole into a world of neverending problems.

**Reply by volfpeter** [reply]
Sadly for some software the choice is to set up a Windows VM or install the application using Wine... In an ideal world, I would agree with you, but it's not as simple as that.
But as I wrote yesterday, I understand the reasons, case closed.

**Comment by volfpeter** [comment]
Well, this is quite a basic thing, I can imagine plenty of people would be interested, so an "official" recommendation/guide would be nice, but of course I understand. Maybe the TheLinuxExperiment channel would be the best qualified to do this :)
BTW: any good content will be welcome!
